api/serverpb/server.proto

```diff
@@ -122,6 +122,8 @@ message Settings {
   string platform_email = 10;
   // DBaaS enabled
   bool dbaas_enabled = 11;
+  // PMM Server public address.
+  string pmm_public_address = 13;
```

Comment:
Number 12 skipped because of consistence with Change Settings?

Reply:
Will update before merge